### PR TITLE
Image: Update Rich Image Icons

### LINKED ISSUE #22570: Update image editing icons
Follow-up from https://github.com/WordPress/gutenberg/pull/21024#issuecomment-632567310

The old icons can be replaced. Switch portrait/landscape isn't included because it will be removed in #22567.

#### Crop

Screenshot 2020-05-22 at 09 05 37

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17.5 7v8H19V7c0-1.1-.9-2-2-2H9v1.5h8c.3 0 .5.2.5.5zM7 17.5c-.3 0-.5-.2-.5-.5V1H5v4H1v1.5h4V17c0 1.1.9 2 2 2h10.5v4H19v-4h4v-1.5H7z"/></svg>
```

#### Rotate left and right

Screenshot 2020-05-22 at 09 19 55

**Rotate Left**

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 4V2.2L9 4.8l3 2.5V5.5c3.6 0 6.5 2.9 6.5 6.5 0 2.9-1.9 5.3-4.5 6.2v.2l-.1-.2c-.4.1-.7.2-1.1.2l.2 1.5c.3 0 .6-.1 1-.2 3.5-.9 6-4 6-7.7 0-4.4-3.6-8-8-8zm-7.9 7l1.5.2c.1-1.2.5-2.3 1.2-3.2l-1.1-.9C4.8 8.2 4.3 9.6 4.1 11zm1.5 1.8l-1.5.2c.1.7.3 1.4.5 2 .3.7.6 1.3 1 1.8l1.2-.8c-.3-.5-.6-1-.8-1.5s-.4-1.1-.4-1.7zm1.5 5.5c1.1.9 2.4 1.4 3.8 1.6l.2-1.5c-1.1-.1-2.2-.5-3.1-1.2l-.9 1.1z"/></svg>
```

**Rotate Right**

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M15.1 4.8l-3-2.5V4c-4.4 0-8 3.6-8 8 0 3.7 2.5 6.9 6 7.7.3.1.6.1 1 .2l.2-1.5c-.4 0-.7-.1-1.1-.2l-.1.2v-.2c-2.6-.8-4.5-3.3-4.5-6.2 0-3.6 2.9-6.5 6.5-6.5v1.8l3-2.5zM20 11c-.2-1.4-.7-2.7-1.6-3.8l-1.2.8c.7.9 1.1 2 1.3 3.1L20 11zm-1.5 1.8c-.1.5-.2 1.1-.4 1.6s-.5 1-.8 1.5l1.2.9c.4-.5.8-1.1 1-1.8s.5-1.3.5-2l-1.5-.2zm-5.6 5.6l.2 1.5c1.4-.2 2.7-.7 3.8-1.6l-.9-1.1c-.9.7-2 1.1-3.1 1.2z"/></svg>
```

#### Flip vertical and horizontal

Screenshot 2020-05-22 at 09 39 41

**Vertical**

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M2 11.2v1.5h20v-1.5H2zM5.5 6c0-.3.2-.5.5-.5h12c.3 0 .5.2.5.5v3H20V6c0-1.1-.9-2-2-2H6c-1.1 0-2 .9-2 2v3h1.5V6zm2 14h2v-1.5h-2V20zm3.5 0h2v-1.5h-2V20zm7-1.5V20c1.1 0 2-.9 2-2h-1.5c0 .3-.2.5-.5.5zm.5-2H20V15h-1.5v1.5zM5.5 18H4c0 1.1.9 2 2 2v-1.5c-.3 0-.5-.2-.5-.5zm0-3H4v1.5h1.5V15zm9 5h2v-1.5h-2V20z"/></svg>
```

**Horizontal**

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 6v12c0 1.1.9 2 2 2h3v-1.5H6c-.3 0-.5-.2-.5-.5V6c0-.3.2-.5.5-.5h3V4H6c-1.1 0-2 .9-2 2zm7.2 16h1.5V2h-1.5v20zM15 5.5h1.5V4H15v1.5zm3.5.5H20c0-1.1-.9-2-2-2v1.5c.3 0 .5.2.5.5zm0 10.5H20v-2h-1.5v2zm0-3.5H20v-2h-1.5v2zm-.5 5.5V20c1.1 0 2-.9 2-2h-1.5c0 .3-.2.5-.5.5zM15 20h1.5v-1.5H15V20zm3.5-10.5H20v-2h-1.5v2z"/></svg>
```

#### Aspect ratio

Screenshot 2020-05-22 at 09 45 38

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18.5 5.5h-13c-1.1 0-2 .9-2 2v9c0 1.1.9 2 2 2h13c1.1 0 2-.9 2-2v-9c0-1.1-.9-2-2-2zm.5 11c0 .3-.2.5-.5.5h-13c-.3 0-.5-.2-.5-.5v-9c0-.3.2-.5.5-.5h13c.3 0 .5.2.5.5v9zM6.5 12H8v-2h2V8.5H6.5V12zm9.5 2h-2v1.5h3.5V12H16v2z"/></svg>
```

### PR DESCRIPTION
**Pull request by ajlende**
## Description

Updates the icons in rich-image with the new ones from #22593 

## How has this been tested?

Add image block, view new edit icons, also check crop.

## Screenshots 

image

image

image

image

You can ignore the change aspect icon since it is being removed in #22817 

## Types of changes

Fixes #22570 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/block-library/src/rich-image/rich-image/index.js [hunk 1]
```diff
@@ -31,14 +39,6 @@ import { compose } from '@wordpress/compose';
...
  */
 import richImageRequest from './api';
-import {
-	RotateLeftIcon,
-	RotateRightIcon,
-	FlipHorizontalIcon,
-	FlipVerticalIcon,
-	CropIcon,
-	AspectIcon,
-} from './icon';
```
Comment (ellatrix) [inline packages/block-library/src/rich-image/rich-image/index.js:41]: Can this file be removed?

Comment (ajlende) [reply packages/block-library/src/rich-image/rich-image/index.js:41]: Sure can! 🙂

Comment (ellatrix) [reply packages/block-library/src/rich-image/rich-image/index.js:41]: Thanks!

**Review by ellatrix — APPROVED** [review]
Let's rebase and merge

**Pull request merged**